source/fuzz/transformation_add_constant_boolean.cpp

```diff
@@ -21,8 +21,8 @@ namespace spvtools {
 namespace fuzz {
 
 TransformationAddConstantBoolean::TransformationAddConstantBoolean(
-    const protobufs::TransformationAddConstantBoolean& message)
-    : message_(message) {}
+    protobufs::TransformationAddConstantBoolean message)
+    : message_(std::move(message)) {}
```

Comment:
Nice. IDE highlights similar changes in all transformations. Additionally, we can get rid of destructors in all fuzzer passes (or mark them with `override`).

Reply:
Yeah - it was the IDE that alerted me to these as I worked on these classes. We should do a full sweep for both of those ideas. Getting rid of destructors sounds good - though I'm not that experienced with when you do and do not need destructors when using inheritance in C++.

Reply:
I think you only need a virtual destructor in the base class. If that destructor is `default`, you don't need to override it in the derived class (unless the one in the derived class won't be `default`).

Also, there is a good description of all of this at https://en.cppreference.com/w/cpp/language/rule_of_three (see `Rule of Zero` at the bottom for polymorphic classes).

Reply:
Thanks for the pointer. Let's do those in separate PRs, though. Are you OK to approve this one?

Reply:
Approved.